### PR TITLE
ci: test empty `PLACEOS_TAG`

### PR DESCRIPTION
**Pull request by caspiano**
Adds a test for systems without `PLACEOS_TAG` defined in its environment

### CODE REVIEW

**Review by kimburgess — COMMENTED** [review]
This is testing for an empty env var, rather than the lack of env var.

If the unset case is what's required, I think this will need a [conditional step](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepsif) with the different env setup instead.

**Comment by caspiano** [comment]
Addressed unset case in latest commit, @kimburgess

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -26,7 +27,13 @@ jobs:
...
     steps:
       - uses: actions/checkout@v2
-      - name: Spin up environment
+      - name: Test initialization, with unset PLACEOS_TAG
+        if: ${{ matrix.version == '' }}
+        run: ./placeos start --verbose
+        env:
+          PLACE_EMAIL: robot@place.tech
+          PLACE_PASSWORD: development
+      - name: Test initialization
```
Comment (kimburgess) [inline .github/workflows/ci.yml:36]: This step also needs a conditional, otherwise the init is happening twice.

**Review by kimburgess — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (caspiano) [reply .github/workflows/ci.yml:36]: I'll spin it into another job

Comment (caspiano) [reply .github/workflows/ci.yml:36]: Curse the lack of anchors.. so much duplication. 
The idea here was to try to test both unset and empty

**Review by kimburgess — COMMENTED** [review]
To untangle this a little, is it possible to split this into two discrete jobs declarations
1. tests env spinup using the `./placeos` CLI
2. tests operation via `docker compose` directly

NB each will need the matrix versions explicitly specified as [YAML anchors are not currently supported](https://github.com/actions/runner/issues/1182) :(

**Comment by caspiano** [comment]
@kimburgess I don't think it's worth the duplication, the test results are pretty clear

**Comment by kimburgess** [comment]
Ah yes, ignore the above. I saw the `docker-compose down` usage and assumed that both scenarios were being tested, rather than just using that for teardown.

**Review by kimburgess — APPROVED** [review]

**Pull request merged**